### ISSUE TITLE
Support for detecting node installed via Volta

### ISSUE THREAD

**Issue by pankajpatel** [sublimelsp/LSP-eslint#63]
Hello,

I use [Volta](https://volta.sh/) to manage the Node environment. Is there a good way to support the Node installed via Volta?

**Comment by rchl** [comment]
Not really, Volta is current-directory based and that doesn't translate that well to ST and its packages.

Also some servers (LSP-pyright for example) are node-based but are not used in JS/Node projects so how would it work then?

You could just use the Node instance managed by lsp_utils.

**Comment by pankajpatel** [comment]
Seems doable, but I was trying to avoid installing multiple copies of Nodejs on the computer.

Thanks for the quick response.
